experimental/dds/tree/src/interface-prototype/Examples.ts

```diff
@@ -0,0 +1,141 @@
+import { v4 } from 'uuid';
+// This file uses these as opaque id types:
+// the user of these APIs should not know or care if they are short IDs or not, other than that they must be converted to StableId if stored for use outside of the shared tree it was acquired from.
+// In practice, these would most likely be implemented as ShortId numbers.
+import { Definition, TraitLabel } from '../Identifiers';
+import { Side } from '../Snapshot';
+import { StableId, TreeNodeData } from './Anchors';
+import {
+	anchorDataFromNodeId,
+	Command,
+	CommandContext,
+	CommandId,
+	CommandRegistry,
+	PrefetchFilter,
+	root,
+	SharedTree,
+	TreeNodeViewReadonly,
+} from './Checkout';
+import { Place, TreeNode } from './MutableAnchors';
+
+//////////////// Command examples //////////////
+
+// Inserts a node with the specified Definition and identifier at the specified Place, and return it.
+export const insertExample = {
+	id: 'f73f004c-3b3e-42fe-b7c9-2e5e8793ca45' as CommandId,
+	run: (
+		context: CommandContext,
+		{ definition: def, identifier: id }: { definition: StableId; identifier: StableId },
+		{ place }: { place: Place }
+	): TreeNode => {
+		const definition = context.loadDefinition(def);
```

Comment:
Feels very odd (and tedious) having to load a definition or an id in a context. I would have thought those were invariant across contexts.